README.md

```diff
@@ -147,7 +147,7 @@ ios_unit_test(
     name = "Tests",
     bundle_id = "com.example.tests",
     minimum_os_version = "15.0",
-    test_host = "//App",
+    test_host = ":App",
```

Comment:
This is valid, and we should also add the new visibility changes to the `App` target as well.